packages/connectivity_plus_linux/lib/src/network_manager.dart

```diff
@@ -60,7 +60,7 @@ class NMConnection extends DBusRemoteObject {
           (value) => (value as DBusString).value,
           onError: (error) => print(error),
         )
-        .then((value) => value ?? '');
+        .then((value) => value);
```

Comment:
same here, we need to keep the fallback value because it would be null if the error handler is hit